### ISSUE TITLE
Global CO2 model run

### ISSUE THREAD

**Issue by ast222340** [geoschem/geos-chem#2305]
### Name and Institution (Required)

Name: Subhadeep Ghosh
Institution: IIT Delhi

### Confirm you have reviewed the following documentation

- [ Y ] [Support guidelines](https://geos-chem.readthedocs.io/en/stable/help-and-reference/SUPPORT.html)
- [ Y ] [User manuals](https://geos-chem.readthedocs.io/en/stable/geos-chem-shared-docs/supplemental-guides/related-docs.html)
- [ Y ] [Debugging GEOS-Chem and HEMCO errors](https://geos-chem.readthedocs.io/en/stable/geos-chem-shared-docs/supplemental-guides/debug-guide.html)
- [ Y ] [Current and past Github issues](https://github.com/geoschem/geos-chem/issues)

### Description of your issue or question

Respected sir,
I am running the global CO2 model. Model version is 14.3.1 and HEMCO version is 3.8.1.I encountered an error.It is not clear to me what error means.
```console
GEOS-Chem ERROR: Error encountered while registering 
State_Diag%BudgetWetDepFull; number of tags is inconsistent with array size
 -> at Register_DiagField_R8_3D (in Headers/state_diag_mod.F90)
Another thing I noticed is that, In HISTORY.rc file
COLLECTIONS: 'Restart',
             'CO2,
             'SpeciesConc',
             'Budget',
```
When i turn on the 'Budget' option the model crashes.Budget corresponding if i comment out the four options below, then the model is running,but why should it be like this?It would be great to solve this problem.
```console
                       'BudgetWetDepFull_?WET?               ',
                       'BudgetWetDepTrop_?WET?               ',
                       'BudgetWetDepPBL_?WET?                ',
                       'BudgetWetDepLevs1to35_?WET?       ',
```                      
[geoschem_config.txt.log](https://github.com/geoschem/geos-chem/files/15451700/geoschem_config.txt.log)
[HISTORY.rc.txt.log](https://github.com/geoschem/geos-chem/files/15451703/HISTORY.rc.txt.log)
[HEMCO_Config.txt.log](https://github.com/geoschem/geos-chem/files/15451704/HEMCO_Config.txt.log)
[GC.txt.log](https://github.com/geoschem/geos-chem/files/15451707/GC.txt.log)
[pbs_job.txt.log](https://github.com/geoschem/geos-chem/files/15451710/pbs_job.txt.log)

**Comment by lizziel** [comment]
Hi @ast222340, wet deposition is not enabled in the CO2 simulation. The budget diagnostics that are causing it to crash and therefore never initialized. This is a bug in the HISTORY.rc template for the carbon simulation which we will fix. Thanks for bringing this to our attention. 

@yantosca

**Comment by yantosca** [comment]
Thank you @lizziel and @ast222340.  We can fix this in a 14.4.1 release.  Right now we are preparing for the IGC11 meeting in June so it may have to wait a bit.

**Comment by ast222340** [comment]
Thank you @lizziel, Now i will switch off the wet deposition part and run the model.Am i right please confirm me.
                       'BudgetWetDepFull_?WET?               ',
                       'BudgetWetDepTrop_?WET?               ',
                       'BudgetWetDepPBL_?WET?                ',
                       'BudgetWetDepLevs1to35_?WET?       ',
Thank you @yantosca, It's great if you look at this after IGC11 meeting is over.

**Comment by lizziel** [comment]
Correct. I will leave this issue open until we put in a fix.

**Comment by yantosca** [comment]
@ast222340 @lizziel: I will close out this issue as we have merged the fix in PR #2313 into the "no-diff-to-benchmark" branch.
